openstax/settings/dev.py

```diff
@@ -11,3 +11,13 @@
     from .local import *
 except ImportError:
     pass
+
+##################################
+#   OVERRIDE ACCOUNTS SETTINGS   #
+##################################
+
+# use default loging and logout urls,
+# Needed for selenium test.
+ACC_APP_LOGIN_URL = None
```

Comment:
I need to override these settings for the selenium tests I wrote.  Which is why I thought it was appropriate to put these settings here.